build_catalog.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+num_commits=$(git rev-list $(git rev-list --max-parents=0 HEAD)..HEAD --count)
+current_commit=$(git rev-parse --short=7 HEAD)
+version="0.1.$num_commits-$current_commit"
+opm_version="1.14.0"
```

Comment:
we may this to be a var that can be set externally with a default

Reply:
I thought about that, but I'm not sure we want a bunch of different opm versions in use.  For now I want to keep as is and we can iterate to determine how to handle the version.

Reply:
ok